.circleci/config.yml

```diff
@@ -5,28 +5,42 @@ jobs:
       - image: circleci/python:3.7.1
         environment:
           PIPENV_VENV_IN_PROJECT: true
+          TERM: linux
```

Comment:
what is that?

Reply:
https://stackoverflow.com/questions/9485699/setupterm-could-not-find-terminal-in-python-program-using-curses/21571407#21571407